apps/web/src/Providers.tsx

```diff
@@ -60,7 +60,7 @@ const Providers: React.FC<PropsWithChildren<{}>> = ({ children }) => {
           </BrowserRouter>
         </QueryClientProvider>
       </SegmentProvider>
-    </ColorSchemeProvider>
+    </ThemeProvider>
```

Comment:
ThemeProvider now lives at the top of the tree.

Reply:
💯